### DISCUSSION TITLE
How can I use the payload dump file?

### DISCUSSION THREAD

**Discussion by SeanWilliamson2004** [rfennell/AzurePipelines#1236] [Q&A] [answered]
I'm using the console interface to produce release notes for a backlog of releases.   As a result it takes some time to process but I want to amend the handlebars template on a trial and error basis.  The payload file seems to be there to allow me to cut out the initial processing when amending the template but I'm struggling to work out how I use it.

Thank you for a great tool and in particular for introducing the 'last successful release' logic.

**Comment by rfennell** [comment, marked as answer]
I think you misunderstand the way the local tester currently works. At present, the `payload.json` is generated on each run (as a reference for you to read to find the data relations).

What you suggested, using a pre-existing `payload.json` without making all the API calls is a great idea. I will add it to the backlog

**Reply by rfennell** [reply]
This feature has been added, see the new usage [here](https://github.com/rfennell/AzurePipelines/tree/main/Extensions/XplatGenerateReleaseNotes/V3/testconsole)
